x-pack/plugins/observability_solution/apm/public/components/app/service_inventory/multi_signal_inventory/table/get_service_columns.tsx

```diff
@@ -0,0 +1,170 @@
+import { i18n } from '@kbn/i18n';
+import { TypeOf } from '@kbn/typed-react-router-config';
+import React from 'react';
+import {
+  asDecimalOrInteger,
+  asMillisecondDuration,
+  asPercent,
+  asTransactionRate,
+} from '../../../../../../common/utils/formatters';
+import { EntityServiceListItem, SignalTypes } from '../../../../../../server/routes/entities/types';
+import { Breakpoints } from '../../../../../hooks/use_breakpoints';
+import { unit } from '../../../../../utils/style';
+import { ApmRoutes } from '../../../../routing/apm_route_config';
+import { EnvironmentBadge } from '../../../../shared/environment_badge';
+import { ServiceLink } from '../../../../shared/links/apm/service_link';
+import { ListMetric } from '../../../../shared/list_metric';
+import { ITableColumn } from '../../../../shared/managed_table';
+import { NotAvailableApmMetrics } from '../../../../shared/not_available_apm_metrics';
+import { TruncateWithTooltip } from '../../../../shared/truncate_with_tooltip';
+import { ServiceInventoryFieldName } from './multi_signal_services_table';
+
+export function getServiceColumns({
+  query,
+  breakpoints,
+  link,
+}: {
+  query: TypeOf<ApmRoutes, '/services'>['query'];
+  breakpoints: Breakpoints;
+  link: any;
+}): Array<ITableColumn<EntityServiceListItem>> {
+  return [
+    {
+      field: ServiceInventoryFieldName.ServiceName,
+      name: i18n.translate('xpack.apm.multiSignal.servicesTable.nameColumnLabel', {
+        defaultMessage: 'Name',
+      }),
+      sortable: true,
+      render: (_, { serviceName, agentName }) => (
+        <TruncateWithTooltip
+          data-test-subj="apmServiceListAppLink"
+          text={serviceName}
+          content={
+            <EuiFlexGroup gutterSize="s" justifyContent="flexStart">
+              <EuiFlexItem grow={false}>
+                <ServiceLink serviceName={serviceName} agentName={agentName} query={query} />
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          }
+        />
+      ),
+    },
+    {
+      field: ServiceInventoryFieldName.Environments,
+      name: i18n.translate('xpack.apm.multiSignal.servicesTable.environmentColumnLabel', {
+        defaultMessage: 'Environment',
+      }),
+      sortable: true,
+      width: `${unit * 9}px`,
+      dataType: 'number',
+      render: (_, { environments }) => {
+        return <EnvironmentBadge environments={environments} />;
+      },
+      align: RIGHT_ALIGNMENT,
+    },
+    {
+      field: ServiceInventoryFieldName.Latency,
+      name: i18n.translate('xpack.apm.multiSignal.servicesTable.latencyAvgColumnLabel', {
+        defaultMessage: 'Latency (avg.)',
+      }),
+      sortable: true,
+      dataType: 'number',
+      align: RIGHT_ALIGNMENT,
+      render: (_, { metrics, signalTypes }) => {
+        return !signalTypes.includes(SignalTypes.METRICS) ? (
+          <NotAvailableApmMetrics />
+        ) : (
+          <ListMetric
+            isLoading={false}
+            hideSeries={true}
```

Comment:
nit: 
```suggestion
            hideSeries
```
and some others below this one